packages/cozy-sharing/src/components/__snapshots__/SharedStatus.spec.js.snap

```diff
@@ -67,7 +67,7 @@ exports[`SharedStatus component should display the link if there is a link 1`] =
               wrapper="div"
             >
               <div
-                className="__react_component_tooltip place-bottom type-dark"
+                className="__react_component_tooltip place-bottom type-dark "
```

Comment:
Why did it change?

Reply:
I think it depends on the nodejs version :/

Reply:
OK. There's nothing blocking, I just wanted to ensure we don't miss something ;)